### ISSUE TITLE
rate limit problems

### ISSUE THREAD

**Issue by Ri0n** [terraform-providers/terraform-provider-launchdarkly#26]
Hi I'm trying to "terraform destroy -parallelism=1" with ~70 flags * 4 environment.
provider version: 1.3.0

It gives me a lot of errors like 
```
Error: failed to update flag "my.flag.name" in project "projectname", environment "dev": 429 Too Many Requests: {"code":"rate_limited","message":"You've exceeded the API rate limit. Try again later."}
```

Is there a way to slowdown the provider even more?

**Comment by ldhenry** [comment]
Hey @Ri0n,

Which version of the provider are you running?

I ask because I added some code to handle rate limits in version 1.3.0 of the provider. Hopefully that solves your issue, but if not, please let me know.

Thanks,
Henry

**Comment by Ri0n** [comment]
It's 1.3.0

**Comment by ldhenry** [comment]
OK thanks for checking. I'll do an investigation into why you are seeing this issue for `terraform destroy`

**Comment by ldhenry** [comment]
I just released v1.3.1 that includes a patch for this but I'm still uncertain if this will solve this issue completely. Please re-open this issue and let me know if if your are still experiencing rate limit issues.

Thanks,
Henry

**Comment by Ri0n** [comment]
Thanks Henry.
I'll test it soon.
